mm2src/adex_cli/src/adex_proc/command.rs

```diff
@@ -8,10 +7,11 @@ where
     T: Serialize + Sized,
 {
     #[serde(flatten, skip_serializing_if = "Option::is_none")]
-    flatten_data: Option<T>,
-    userpass: String,
+    pub(self) flatten_data: Option<T>,
```

Comment:
no need to use `pub(self)` https://github.com/KomodoPlatform/atomicDEX-API/pull/1768#discussion_r1236008159

Reply:
OMG! Sure, I missed it
[done](https://github.com/KomodoPlatform/atomicDEX-API/pull/1889/commits/6f317047772adf92ed46b167d34d0d7316928766)